manager/apis/app/v1alpha1/m4dapplication_types.go

```diff
@@ -57,11 +58,17 @@ type ExternalDataContext struct {
 	// +optional
 	Recurrence RecurrenceType `json:"recurrence,omitempty"`
 }
+*/
 
 // DataContext indicates data set chosen by the Data Scientist to be read from or written to by his application,
 // and includes information about the data format and technologies used by the application
 // to access the data.
 type DataContext struct {
+	// DataCatalogID indicates the data catalog that contains the information about the data set.  If no value is provided the assumption is there is only one data catalog.
+	// TODO: Temporary pending design for multi-catalog support.
+	// +optional
+	DataCatalogID string `json:"dataCatalogID"`
+
```

Comment:
Why optional? Manager needs to know what connector/catalog to access. Also, this should be an enum rather than a free string, e.g. Egeria, inline, ...

Reply:
@shlomitk1 Agreed.  I'm waiting for the "inline" catalog design details before finalizing this.  @roee88 is working on this I believe.

Reply:
Removed